workflow/tests/base_results/results_workflow_simulations1.csv

```diff
@@ -183,10 +183,10 @@ base-hvac-air-to-air-heat-pump-1-speed-seer2-hspf2.xml,45.873,45.873,45.873,45.8
-base-hvac-air-to-air-heat-pump-var-speed-backup-furnace.xml,54.261,54.261,39.013,39.013,15.248,0.0,0.0,0.0,0.0,0.0,5.145,0.36,0.0,0.46,2.418,0.478,9.014,0.0,0.0,4.507,0.0,0.334,0.0,0.0,0.0,0.0,2.072,0.0,0.0,0.319,0.365,1.513,1.529,0.0,2.116,8.384,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,15.248,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,29.44,12.658,16.007,9.071,0.615,0.0,0.0,2.0,11.0,3138.7,3319.7,3319.7,26.783,17.748,0.0,3.252,3.64,0.512,7.511,0.63,10.086,-12.696,0.0,0.0,0.0,8.302,-0.064,4.804,0.0,0.727,0.0,12.639,-7.798,-2.488,0.0,-0.127,-0.439,-0.048,2.748,-0.02,-1.332,11.717,0.0,0.0,0.0,-6.237,-0.059,-1.139,-3.789,-0.162,0.0,5.208,7.783,2.019,1354.7,998.0,11171.6,2563.5,0.0,18000.0,18000.0,60000.0,6.8,91.76,40224.0,16125.0,7508.0,0.0,575.0,6840.0,0.0,0.0,2408.0,2171.0,4597.0,19783.0,6042.0,7037.0,0.0,207.0,265.0,0.0,0.0,0.0,2293.0,619.0,3320.0,144.0,0.0,-656.0,800.0
+base-hvac-air-to-air-heat-pump-var-speed-backup-boiler-hvac-seasons.xml,54.151,54.151,38.777,38.777,15.374,0.0,0.0,0.0,0.0,0.0,5.322,0.38,0.0,0.058,2.39,0.477,9.014,0.0,0.0,4.507,0.0,0.334,0.0,0.0,0.0,0.0,2.07,0.0,0.0,0.319,0.365,1.513,1.529,0.0,2.116,8.384,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,15.374,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,28.976,11.489,15.75,9.071,0.616,0.0,0.0,1.0,13.0,3140.5,2982.9,3140.5,23.257,17.755,0.0,3.264,3.597,0.506,7.506,0.614,9.915,-12.591,0.0,0.0,0.0,8.221,-0.031,5.815,0.0,0.719,0.0,11.235,-7.688,-2.467,0.0,-0.16,-0.467,-0.052,2.794,-0.032,-1.455,11.822,0.0,0.0,0.0,-6.255,-0.027,-1.459,-3.737,-0.167,0.0,5.139,7.892,2.04,1354.7,998.0,11171.5,2563.5,0.0,18000.0,18000.0,60000.0,6.8,91.76,31615.0,7517.0,7508.0,0.0,575.0,6840.0,0.0,0.0,2408.0,2171.0,4597.0,19783.0,6042.0,7037.0,0.0,207.0,265.0,0.0,0.0,0.0,2293.0,619.0,3320.0,144.0,0.0,-656.0,800.0
```

Comment:
Differences here are because? Now calculating pressure from elevation, something like that?

Reply:
I'm not immediately sure. This is the last thing on my list to look into.

Reply:
Ah, it's caused by rounding the default elevation to one decimal place:
https://github.com/NREL/OpenStudio-HPXML/blob/16071322ddadae284de41cbcb190e245ec1efb51/HPXMLtoOpenStudio/resources/hpxml_defaults.rb#L566

When I remove the `.round(1)`, the results are identical to master. It seems entirely reasonable to me to round the elevation to 0.1 ft (~1 inch).